pkg/orchestrator/etcd_worker.go

```diff
@@ -119,9 +120,8 @@ func (worker *EtcdWorker) initMetrics(captureAddr string) {
 // A tick is generated either on a timer whose interval is timerInterval, or on an Etcd event.
 // If the specified etcd session is Done, this Run function will exit with cerrors.ErrEtcdSessionDone.
 // And the specified etcd session is nil-safety.
-func (worker *EtcdWorker) Run(ctx context.Context, session *concurrency.Session, timerInterval time.Duration, captureAddr string) error {
+func (worker *EtcdWorker) Run(ctx cdcContext.Context, session *concurrency.Session, timerInterval time.Duration, captureAddr string) error {
```

Comment:
Could you pass role explicitly? So that we don't need to add a new "global" variable in context.